apps/docs/pages/guides/integrations/estuary.mdx

```diff
@@ -72,21 +70,25 @@ A form appears with the properties required for a Postgres materialization.
 
 2. Choose a unique name for your materialization like you did when naming your capture; for example, `acmeCo/mySupabaseMaterialization`.
 
-3. Fill out the required properties for PostgreSQL. You can find most of these in Supabase by going to the **Settings** section and clicking **Database**.
+3. Fill out the required properties for Postgres. You can find most of these in Supabase by going to the **Settings** section and clicking **Database**.
 
    - **Address**: Format at `<host>:<port>`.
    - **User**: Usually, this is `postgres`.
    - **Password**: The password you set when you created your Supabase project.
```

Comment:
Workaround for the broken numbering when an unordered list is nested within an ordered list:
```suggestion
   **Address**: Format at `<host>:<port>`.
   
   **User**: Usually, this is `postgres`.
   
   **Password**: The password you set when you created your Supabase project.
```